src/additional_tools/expression_manipulation.jl

```diff
@@ -0,0 +1,125 @@
+function createemptyexpression!(EP::Model, exprname::Symbol, dim1::Int64)
+	EP[exprname] = Vector{AffExpr}(undef, dim1)
+	for i=1:dim1
+		EP[exprname][i] = AffExpr(0.0)
+	end
+end
+
+function createemptyexpression!(EP::Model, exprname::Symbol, dim1::Int64, dim2::Int64)
+	EP[exprname] = Matrix{AffExpr}(undef, (dim1, dim2))
+	for j=1:dim2
+		for i=1:dim1
+			EP[exprname][i,j] = AffExpr(0.0)
+		end
+	end
+end
```

Comment:
Please prefer spaces to tabs when writing new GenX functions. Indentation should be four spaces. This is to be consistent with existing GenX code.